test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -819,6 +820,74 @@ ava('._prependLegCollectionWithRouteAmendment() adds LegModels for each routeStr
     t.true(fms.legCollection[1].routeString === routeAmmendmentMock[1]);
 });
 
+ava('._regenerateSidLeg() creates a new SID leg of identical route string for the currently assigned departure runway', (t) => {
+    const initialRunwayMock = airportModelFixture.getRunway('19L');
+    const nextRunwayMock = airportModelFixture.getRunway('19R');
+    const fms = buildFmsMockForDeparture();
+    const replaceDepartureProcedureSpy = sinon.spy(fms, 'replaceDepartureProcedure');
+    const sidLegIndex = fms._findLegIndexForProcedureType(PROCEDURE_TYPE.SID);
+    const oldWaypoints = fms.legCollection[sidLegIndex].waypointCollection;
+
+    t.true(_isEqual(fms.departureRunwayModel, initialRunwayMock));
```

Comment:
Correct, the idea being that the test only gives a good result if we know the first value of the `departureRunwayModel` is one value which is changed to the specified value. I could skip the verification on the first part, but kinda thought it provided protection in case the fixtures were ever changed. For example, in this case, if the `buildFmsMockForDeparture()` was changed to use `19R` by default, this test would pass, even if the method was broken. I was trying to forcibly show that the value was changed between two expected values. Easily could be seen as over-defensive though.

Think it's better to just check the "after" instead?